### ISSUE TITLE
Dashboard Alerts

### ISSUE THREAD

**Issue by ztimson** [swarmpit/swarmpit#419]
# Feature Request
I have a spare monitor that I leave the dashboard running on in my office so I can monitor my servers at a glance. It would be extremely helpful if the dashboard would show an [alert banner](https://getbootstrap.com/docs/4.0/components/alerts/) at the top of the page when important events occur. i.e.:
 - node is down (error)
 - node is draining (warn)
 - node is paused (warn)
 - task failed (error)
 - task updating (warn)

**Comment by Doflatango** [comment]
It would be very helpful if swarmpit can send those events to a  "webhook", therefore, we can keep track of the service status.

**Comment by ztimson** [comment]
Agreed, that would open up the possibility of having slack/discord/team notifications on events

**Comment by nohaapav** [comment]
We're planning to integrate slack ..

**Comment by Doflatango** [comment]
> We're planning to integrate slack ..

:sweat_smile:

It's enough just leave a webhook, it will be more flexible. Actually some critical event i will send by SMS too.

**Comment by nohaapav** [comment]
@Doflatango of course it's going to be webhook, what else .. smartass :)
